crates/ra_proc_macro/src/process.rs

```diff
@@ -44,8 +44,9 @@ impl Drop for Process {
 }
 
 impl Process {
-    fn run(process_path: &Path) -> Result<Process, io::Error> {
+    fn run<T: AsRef<str>>(process_path: &Path, args: &[T]) -> Result<Process, io::Error> {
```

Comment:
As this wraps the underling `Command`, an appropriate rule-of-thumb would be about not needlessly restricting the interface. It's better to just pass through the `args` exactly as the `.args` function wants:

```
pub fn args<I, S>(&mut self, args: I) -> &mut Command where
    I: IntoIterator<Item = S>,
    S: AsRef<OsStr>, 
```